changelog/3737.deprecation.rst

```diff
@@ -0,0 +1,23 @@
+**RemovedInPytest4Warnings are now errors by default.**
```

Comment:
this should be categorized as breaking change, since the deprecations exist, and now they shoot feet ^^

Reply:
Yeah I was unsure if this note should be a "deprecation" or "removal", will change.

Reply:
Done!